scraper/pipelines.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+# Define your item pipelines here
+#
+# Don't forget to add your pipeline to the ITEM_PIPELINES setting
+# See: http://doc.scrapy.org/en/latest/topics/item-pipeline.html
+
+# based on the Scrapy Tutorial on item pipelines
+from scrapy.exceptions import DropItem
+
+
+class DeDupePipeline(object):
```

Comment:
largely borrowed this class from the tutorial available on the scrapy framework website here: http://doc.scrapy.org/en/latest/intro/tutorial.html